### PR TITLE
Update opentracing.Tag comments

### PR DESCRIPTION
**Pull request by jmacd**

### CODE REVIEW

**Comment by jmacd** [comment]
@dominikh
See if you like these changes.

**Comment by dominikh** [comment]
LGTM

File: ext/tags.go [hunk 1]
```diff
@@ -15,39 +15,37 @@ import opentracing "github.com/opentracing/opentracing-go"
 //
 var (
 	//////////////////////////////////////////////////////////////////////
-	// SpanKind (client/server)
+	// SpanKind (client/server) tag names
```
Comment (bensigelman) [inline ext/tags.go]: same as that other PR -- `s/name/key/` for tags.

**Comment by bensigelman** [comment]
(otherwise LGTM...)

File: ext/tags.go [hunk 2]
```diff
@@ -83,22 +81,34 @@ var (
...
+	// Error Tag name
 	//////////////////////////////////////////////////////////////////////
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
 )
 
+var (
+	//////////////////////////////////////////////////////////////////////
+	// Conventional SpanKind Tags
```
Comment (yurishkuro) [inline ext/tags.go]: what does 'conventional' mean here?

File: ext/tags.go [hunk 3]
```diff
@@ -83,22 +81,34 @@ var (
...
+	//////////////////////////////////////////////////////////////////////
+
+	// TagSpanKindRPCClient is a tag indicating the span is a RPC client.
+	TagSpanKindRPCClient = opentracing.Tag{Key: string(SpanKind), Value: SpanKindRPCClientTagValue}
+
+	// TagSpanKindRPCServer is a tag indicating the span is a RPC client.
+	TagSpanKindRPCServer = opentracing.Tag{Key: string(SpanKind), Value: SpanKindRPCServerTagValue}
```
Comment (yurishkuro) [inline ext/tags.go]: what do we lose if we just call it SpanKindRPCServer?  It matches with the other tags in this file, e.g.
```go
ext.SamplingPriority.Set(span, 1)
ext.SpanKindRPCClient.Set(span) // value implied
```

**Comment by bensigelman** [comment]
@jmacd can you either update XOR close this PR? Thanks.

File: ext/tags.go (hunk 2 above)
Comment (jmacd) [inline ext/tags.go]: To mean "conventional" means the same thing written above, "common [...] recommended for better portability [...]"

File: ext/tags.go (hunk 1 above)
Comment (jmacd) [inline ext/tags.go]: OK

File: ext/tags.go (hunk 3 above)
Comment (jmacd) [inline ext/tags.go]: I removed "Tag" as you suggested.

**Comment by jmacd** [comment]
Hey all, sorry for the long delay. I'll submit this and have applied changes for all the recommendations above.

**Comment by yurishkuro** [comment]
One way to keep this backwards compatible is to alias the old constant names to new constants.

**Comment by jmacd** [comment]
I'm officially judging this not worth the trouble.

**Pull request closed without merging**